src/cpu/aarch64/acl_reorder.hpp

```diff
@@ -22,6 +22,28 @@
 #include "cpu/aarch64/cpu_isa_traits.hpp"
 #include "cpu/reorder/cpu_reorder_pd.hpp"
 
+namespace {
+    int do_transpose(const dnnl::impl::memory_desc_t* src_md,const dnnl::impl::memory_desc_t* dst_md) {
+        auto src_md_ = dnnl::impl::md2fmt_tag_str(src_md);
```

Comment:
This is very clever, but if I'm honest it does make me a bit nervous to use format strings like this.